src/pages/home/report/ReportActionCompose.js

```diff
@@ -222,15 +239,34 @@ class ReportActionCompose extends React.Component {
     }
 
     /**
-     * Listens for the keyboard shortcut and submits
-     * the form when we have enter
+     * Listens for keyboard shortcuts and applies the action
      *
      * @param {Object} e
      */
-    triggerSubmitShortcut(e) {
-        if (e && e.key === 'Enter' && !e.shiftKey) {
-            e.preventDefault();
-            this.submitForm();
+    triggerShortcut(e) {
+        if (e) {
+            // Submit the form when Enter is pressed
+            if (e.key === 'Enter' && !e.shiftKey) {
+                e.preventDefault();
+                this.submitForm();
+            }
+
+            // Trigger the edit box for last sent message if ArrowUp is pressed
+            if (e.key === 'ArrowUp' && this.state.isCommentEmpty) {
+                e.preventDefault();
+
+                const reportActionKey = _.find(
+                    Object.keys(this.props.reportActions).reverse(),
+                    key => this.props.reportActions[key].actorEmail === this.props.session.email,
```

Comment:
Not sure if this the correct way to do it. You should sort by the `sequenceNumber`.

Reply:
I can see that report actions are being sorted by `sequenceNumber` here:
https://github.com/Expensify/Expensify.cash/blob/9f305176681385f2aa80f43de06fc6dffa7f0b83/src/pages/home/report/ReportActionsView.js#L243-L255

Is this a requirement, however? In my local storage for any `ReportActions_<report_id>` `sequenceNumber` always matches the key of a report action.

Reply:
What do you mean? This code just loops though the `reportActions` backwards and finds the first one where the actor email is the current user.

Reply:
Objects keys are not sequential. so keys' order could be discrete.

Reply:
All the keys in my localstorage are ordered and identical to `sequenceNumber`. If one can't trust this order, I think running `sortBy` on entire object for every action trigger would not be very efficient. It would be better to have some storage for sorted list shared with `ReportActionsView.js` in that case.